jax/nn/__init__.pyi

```diff
@@ -0,0 +1,99 @@
+from typing import Any, List, Literal, Union, overload, Sequence
+
+import jax.numpy as jnp
+from jax._src.typing import (
+    Array, ArrayLike, DTypeLike
+)
+from jax._src.core import AxisName
+from jax._src.cudnn.scaled_matmul_stablehlo import BlockScaleConfig
+from jax._src.lax.lax import DotDimensionNumbers
+
+from jax.nn import initializers as initializers
+
+_Axis = Union[None, int, Sequence[int]]
+
+
+def identity(x: ArrayLike) -> Array: ...
+def relu(x: ArrayLike) -> Array: ...
+def squareplus(x: ArrayLike, b: ArrayLike = 4) -> Array: ...
+def softplus(x: ArrayLike) -> Array: ...
+def sparse_plus(x: ArrayLike) -> Array: ...
+def soft_sign(x: ArrayLike) -> Array: ...
+def sigmoid(x: ArrayLike) -> Array: ...
+def sparse_sigmoid(x: ArrayLike) -> Array: ...
+def silu(x: ArrayLike) -> Array: ...
+def swish(x: ArrayLike) -> Array: ...
+def mish(x: ArrayLike) -> Array: ...
+def log_sigmoid(x: ArrayLike) -> Array: ...
+def elu(x: ArrayLike, alpha: ArrayLike = 1.0) -> Array: ...
```

Comment:
In general the actual default value is not important for the type interface, so instead you should write `alpha: ArrayLike = ...`